pkg/precice-adapter/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "OpenFOAM-preCICE adapter",
```

Comment:
I think i would prefer `OpenFOAM-preCICEAdapter` or - or _ for white spaces.

Reply:
@greole @MakisH I'll go with `precice-adapter` for now. FWIW, the `name` field is not used at all by styro right now, so we can change our minds on that later.